internal/asyncapi/command_export.go

```diff
@@ -275,7 +278,16 @@ func (c *command) getMessageExamples(consumer *ckgo.Consumer, topicName, content
 		return nil, fmt.Errorf("failed to get deserializer for %s", valueFormat)
 	}
 
-	err = deserializationProvider.InitDeserializer("", "", "value", "", "", "", srClient)
+	token, err := auth.GetDataplaneToken(c.Context)
+	if err != nil {
+		return nil, err
+	}
+	srClusterId, srEndpoint, err := c.GetCurrentSchemaRegistryClusterIdAndEndpoint()
+	if err != nil {
+		return nil, err
+	}
+
+	err = deserializationProvider.InitDeserializer(srEndpoint, srClusterId, "value", "", "", token, nil)
```

Comment:
Hopefully we don't need to pass anything for these other two `""` values. It's hard to tell what's going on when a function takes 7 arguments...

Reply:
Lol +1. It's for apikey / secret, alternative to sending token. Should be separate call I guess